### PR TITLE
[improvement](publish version) reduce publish version not exist log

### PR DESCRIPTION
**Pull request by yujun777**
when publish version on not continous mow tablets,  it will try to publish multiple times.   reduce their log.

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yujun777** [comment]
run buildall

**Review by deardeng — APPROVED** [review]
LGTM

File: be/src/olap/task/engine_publish_version_task.cpp [hunk 1]
```diff
@@ -189,6 +191,7 @@ Status EnginePublishVersionTask::finish() {
                     if (tablet->check_version_exist(version)) {
                         continue;
                     }
+                    not_continuous_mow_tablets.insert(tablet_info.tablet_id);
```
Comment (zhannngchen) [inline be/src/olap/task/engine_publish_version_task.cpp:194]: move it to L196, inside `handle_version_not_continuous`

**Comment by yujun777** [comment]
run buildall

File: be/src/olap/task/engine_publish_version_task.cpp (hunk 1 above)
Comment (yujun777) [reply be/src/olap/task/engine_publish_version_task.cpp]: update,  ignore all tablet error when met PUBLISH_VERSION_NOT_CONTINUOUS error

**Review by zhannngchen — APPROVED** [review]
LGTM

**Pull request merged**